### ISSUE TITLE
Mod files not uploaded to curseforge

### ISSUE THREAD

**Issue by C0rn3j** [DrZhark/mocreatures#72]
This makes my updater script not work for this mod.

**Comment by axieum** [comment]
Agreed, plus I cannot add MoCreatures and CustomMobSpawner to my Modpack since it is apparently disallowed as an override.

**Comment by C0rn3j** [comment]
I contacted one of the developers on their discord channel - 

I was met with an extremely hostile response and a ban -

apparently the mod is closed source, will remain closed source, there will be no more uploads on Curseforge and using this mod with any other mod is against the author's will.

Also this issue tracker is a total graveyard because "we have lives".

Removed the mod from my modpack and will not care about it and its authors anymore.

**Comment by axieum** [comment]
That's very sad to hear, how can you develop a mod for a game where people wish to have fun! I'd love to play with animals, and have much more lively aspects in my worlds. Modpacks are growing in popularity, thus I see no reason as to why they shouldn't be jumping on the bandwagon to gain exposure.

I wonder if something happened, there must be a reason as to why they are halting involvement with CurseForge!

**Comment by C0rn3j** [comment]
Likely because they can't earn money through ads on their website if they use CF. 

Could have just asked their userbase for donations but that implies they care about them in the first place.

**Comment by zerotheliger** [comment]
Wow these devs are the worst ive seen since that certain incident who made his stuff incompatable.  This guy needs to learn to get along with other modders... And stop thinking that adfly will get him any notable money.. I find it hilarious thats the main reason why he lost so much money on patreon.

**Comment by zerotheliger** [comment]
And tbh its not like he can stop us from putting his mod on a server with other mods lol might have to figure out a way to asm his mod to fix all the incompatibilities. Or just copy all the mechanics with new code and textures and make a mod that actually wants to work with the community and other mods.

**Comment by axieum** [comment]
If you could create some spin-off of the mod that'd be great. But I fear it might cause legal dispute. That is if you were to use the same models etc. That'd probably be the hardest part, creating all the models for the different animals etc.

**Comment by Kowantify** [comment]
I got banned aswell.. bloodshot is a garbage dev. Acting like he's so important and Minecraft revolves around him
